tests/test_asyncio/test_cache.py

```diff
@@ -2,36 +2,38 @@
 
 import pytest
 import redis.asyncio as redis
+from redis._cache import _LocalCache
 from redis.utils import HIREDIS_AVAILABLE
 
 
 @pytest.mark.skipif(HIREDIS_AVAILABLE, reason="PythonParser only")
 async def test_get_from_cache():
-    r = redis.Redis(cache_enable=True, single_connection_client=True, protocol=3)
+    cache = _LocalCache()
+    r = redis.Redis(protocol=3, client_cache=cache)
```

Comment:
Since we keep doing these, maybe a resp3client helper would be easier, then it can just be passed in repeatedly - similar to _r_